testing/src/cypress/page-objects/student-objects/WelcomeObject.ts

```diff
@@ -17,6 +17,10 @@ export default class WelcomeObject {
     return cy.contains("Sign Up with BCSC");
   }
 
+  loginSignUpButton() {
+    return cy.contains("Login / Sign up with BCSC");
```

Comment:
Not part of the PR, but we will not be using any sort of selectors other than `data-cy` to get the elements from the UI. As needed we can keep adding the `data-cy` on Vue and form.io definitions.